README.md

```diff
@@ -1,37 +1,20 @@
-<p align="center"><img src="docs/img/Thanos-logo_fullmedium.png" alt="Thanos Logo"></p>
```

Comment:
The Thanos logo got removed, was it intended?

Reply:
aiai (: